project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_quickpost.scss

```diff
@@ -1,3 +1,10 @@
+.quickpost-title {
+
+  &:focus::-webkit-input-placeholder {
```

Comment:
@ordavidil Can I go back to use this method? (on the standup this morning IE9 support was dropped)